tests/Agent/IntegrationTests/IntegrationTests/Logging/LocalDecorationTests.cs

```diff
@@ -76,6 +81,10 @@ public void LogIsDecorated()
                 Assert.NotNull(traceId);
                 Assert.NotNull(spanId);
                 Assert.Equal(_primaryApplicationName, entityName);
+
+                // Make sure the original message is there too
+                var commandResults = Regex.Split(testOutput, System.Environment.NewLine).Where(l => !l.Contains("EXECUTING"));
+                Assert.Contains(_testMessage, string.Join(System.Environment.NewLine, commandResults));
```

Comment:
I think this check should be moved out of any if/else logic since we always want to make sure the original log message exists

Reply:
Oh, that's a good catch.  Will do.